addons/html_editor/static/tests/utils/regex.test.js

```diff
@@ -152,5 +152,5 @@ testUrlRegex("abc.abc.com", { insideText: true });
 testNotUrlRegex("abc.abc.abc", { insideText: true });
 testUrlRegex("http://abc.abc.abc", { insideText: true });
 testUrlRegex("https://abc.abc.abc", { insideText: true });
-testUrlRegex("1234-abc.runbot007.odoo.com/web#id=3&menu_id=221", { insideText: true });
-testUrlRegex("https://1234-abc.runbot007.odoo.com/web#id=3&menu_id=221", { insideText: true });
+testUrlRegex("1234-abc.runbot007.odoo.com/odoo/3?menu_id=221", { insideText: true });
+testUrlRegex("https://1234-abc.runbot007.odoo.com/odoo/3?menu_id=221", { insideText: true });
```

Comment:
I will not modify this tests, they test the regex of the URL on the html_editor (we can still copy/past old URLs on the html_editor).
I will introduce two new tests, with the new url.

Reply:
Hello @jpp-odoo 
Suggested changes have been made, please check it.
Thanks :slightly_smiling_face: